README.md

```diff
@@ -33,7 +33,7 @@ Flannel can be added to any existing Kubernetes cluster though it's simplest to
 
 For Kubernetes v1.17+
 ```
-kubectl apply -f https://raw.githubusercontent.com/flannel-io/flannel/master/Documentation/kube-flannel.yml
+kubectl apply -f https://raw.githubusercontent.com/flannel-io/flannel/v0.20.2/Documentation/kube-flannel.yml
```

Comment:
I think it makes more sense to leave it as master, otherwise we will need to change this line in each release, or?

Reply:
We need to change it on each release, but if there are delay on the publishing of the image the documentation is pointing on a not working manifest. I didn't find a way to point to the latest release.

Reply:
What about using the tag `latest` in the manifest?